_source/_posts/2023-03-01-jakarta-ee-oidc.md

```diff
@@ -0,0 +1,497 @@
+---
+layout: blog_post
+title: "Use Jakarta EE 10 with OpenID Connect Authentication"
```

Comment:
This scores a 57 on our title analyzer. 

Other options:

- Use Jakarta EE 10 with OIDC Authentication: 58
- Jakarta EE 10 with OpenID Connect Authentication: 59
- Get Started with Jakarta EE 10 and OpenID Connect Authentication: 55

Reply:
Keyword traffic:

- jakarta ee: 9900
- jakarta security: 170
- jakarta ee 10: 8080

To me, it seems we should optimize for "Jakarta EE" and "Jakarta EE 10" in the title and first few headings.